### PR TITLE
Add category files, fix some capitalization, and add new blog post

### PR DESCRIPTION
**Pull request by DuroCodes**
Adds the blog post about deploying Sern bot with Railway.
I also changed the docusaurus.config.js file so all blog posts display instead of only 5.
Updated some capitalization in titles, such as "transition from v2 to v3" → "Transition from v2 to v3"

### CODE REVIEW

**Comment by jacoobes** [comment]
The sidebar seems to be not working 🤔 
ảnh
It doesn't expand to its subcontents

File: blog/2024-02-23-mdx-blog-post.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+slug: railway-deploy
+title: Deploying Sern with Railway
```
Comment (SrIzan10) [inline blog/2024-02-23-mdx-blog-post.md]: lowercase please (applies to all other lines with the same mistake)

File: docusaurus.config.js
Comment (SrIzan10) [inline docusaurus.config.js:1]: looks like you changed this file, please also edit `docgen.docusaurus.config.js`.

**Review by SrIzan10 — CHANGES_REQUESTED** [review]
there ya go, take a look at these :wink:

**Comment by SrIzan10** [comment]
I think it's ready to merge, but I want @Murtatrxx or @jacoobes to have the last word

**Comment by Murtatrxx** [comment]
Aren't these already planned for the guide update

**Comment by jacoobes** [comment]
the tutorial pr has been stale for a while but i feel like this would be a nice addon. plus i think Duro is working on adding images

**Comment by jacoobes** [comment]
ship it

**Pull request merged**